amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/ExampleEmitters.scala

```diff
@@ -151,9 +152,20 @@ case class NamedMultipleExampleEmitter(key: String,
     extends MultipleExampleEmitter(key, examples, ordering, references) {
 
   def emit(b: PartBuilder) = {
-    val emitters = examples.map(NamedExampleEmitter(_, ordering))
+    val emitters = examplesWithKey.map { case (key, example) => new KeyedExampleEmitter(key, example, ordering) }
     b.obj(traverse(ordering.sorted(emitters), _))
   }
+
+  def examplesWithKey: Seq[(String, Example)] = {
+    val counter  = new IdCounter()
+    val usedKeys = mutable.Set[String]()
+    examples.map { ex =>
+      val name = ex.name.option()
+      val key  = name.filter(!usedKeys.contains(_)).getOrElse(counter.genId("amf_example"))
```

Comment:
why "amf_example" and not "example"?

Reply:
mainly to avoid collisions with actual names defined in the api